components/Press/PressLinks/PressLinks.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './PressLinks.css';
+import * as Links from './Links';
+import LinkGroup from './LinkGroup';
+
+PressLinks.propTypes = {
+  MaxLinks: PropTypes.number,
+};
+
+PressLinks.defaultProps = {
+  MaxLinks: 5,
+};
+
+function PressLinks({ MaxLinks }) {
+  return (
+    <div className={styles.logos}>
+      <div className={styles.flexContainer}>
+        {Object.keys(Links).map(group => (
+          <LinkGroup
+            key={`LinkGroup_${group}`}
```

Comment:
you set the key to `Links[group]` - the urls should always be unique.

Reply:
LinkGroup is the name of the group "Boston, NYC, General" Perhaps using Links is a poor naming option as the Links are one level down yet.

Reply:
Maybe `Object.keys(Articles).map(({ title, link }) => ( <ArticleItem key={link} ... etc`

Reply:
Much better, Ill also change the file name Links to Articles to match.